scripts/gen_lib.lua

```diff
@@ -0,0 +1,71 @@
+lpeg = require('lpeg')
```

Comment:
src/generators/{bin,lib} is fine. nvim is not necessary as it is there for namespacing.